scope_hunter.py

```diff
@@ -817,6 +817,7 @@ def plugin_loaded():
 
 
 def plugin_unloaded():
-    """Kill the thead."""
+    """Kill the thread."""
 
+    pref_settings.clear_on_change('scopehunter_reload')
```

Comment:
`pref_settings` is not a global variable.

Reply:
Oops, fixed.